docs/source/asr/speech_intent_slot/configs.rst

```diff
@@ -88,6 +88,11 @@ An example config for the model can be:
       subsampling_factor: 4 # must be power of 2
       subsampling_conv_channels: -1 # -1 sets it to d_model
 
+      # Reduction params
```

Comment:
Maybe adding a comment what is the purpose of reduction?

Reply:
Added